docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc

```diff
@@ -41,22 +43,27 @@ quarkus.http.auth.permission.permit1.methods=GET
 quarkus.http.auth.permission.deny1.paths=/forbidden                                 <4>
 quarkus.http.auth.permission.deny1.policy=deny
 ----
-<1> This defines a role based policy that allows users with the `user` and `admin` roles. This is referenced by later rules.
-<2> This is a permission set that references the previously defined policy. `roles1` is an arbitrary name, you can call the permission sets whatever you want.
-<3> This permission references the default `permit` built-in policy to allow `GET` methods to `/public`. This is actually a no-op in this example, as this request would have been allowed anyway.
-<4> This permission references the built-in `deny` policy for `/forbidden`. This is an exact path match as it does not end with `*`.
-
-Permissions are defined in config using permission sets. These are arbitrarily named permission grouping. Each permission
-set must specify a policy that is used to control access. There are three built-in policies: `deny`, `permit` and `authenticated`,
-which respectively permits all, denies all and only allows authenticated users.
-
-It is also possible to define role based policies, as shown in the example. These policies will only allow users with the
-specified roles to access the resources.
+<1> This defines a role-based policy that allows users with the `user` and `admin` roles. This is referenced by later rules.
+<2> This is a permission set that references the previously defined policy.
+`roles1` is an arbitrary name; you can call the permission sets whatever you want.
+<3> This permission references the default `permit` built-in policy to allow `GET` methods to `/public`.
+This is a no-op in this example, as this request would have been allowed anyway.
+<4> This permission references the built-in `deny` policy for `/forbidden`.
+This is an exact path match as it does not end with `*`.
+
```

Comment:
Nice idea to summarize all the options, but I'd consider moving this new addition (lines 54-60) above, just below `Authorization using configuration`, before introducing the examples